packages/shared-ui/src/flowHandler/flows/login/utils.ts

```diff
@@ -193,8 +197,12 @@ export const initConditionalUI = async (state: FlowHandlerState): Promise<FlowUp
   return FlowUpdate.ignore();
 };
 
-export const appendPasskey = async (authService: AuthService): Promise<FlowUpdate> => {
-  await authService.appendPasskey();
+export const appendPasskey = async (authService: AuthService, flowOptions: FlowOptions): Promise<FlowUpdate> => {
```

Comment:
we have two implementations of appendPasskey (one for login, one for signUp). While this on takes all flowOptions as a parameter, the other one just takes the retryOnPasskeyError setting. Can't we make them similar?